package.json

```
@@ -13,7 +13,8 @@
     "lint:md": "prettier --ignore-path .gitignore --check '**/*.md'",
     "lint": "run-s --continue-on-error lint:js lint:json lint:md",
     "test": "jest",
-    "checks": "run-s lint test"
+    "checks": "run-s lint test",
+    "postinstall": "./script/postinstall"
```

```suggestion
    "checks": "run-s lint test"
```

As you said: perfectly fine to ditch this.